tests/openedx_learning/apps/authoring/backup_restore/test_backup.py

```diff
@@ -172,33 +183,29 @@ def test_lp_dump_command(self):
 
             # Check the content of the entity TOML files
             for entity in self.all_components:
-                current_draft_version = getattr(entity, "draft", None)
-                current_published_version = getattr(entity, "published", None)
-                expected_content = [
-                    '[entity]',
-                    f'uuid = "{entity.uuid}"',
-                    'can_stand_alone = true',
-                    '[entity.draft]',
-                    f'version_num = {current_draft_version.version.version_num}',
-                    '[entity.published]',
-                ]
-                if current_published_version:
-                    expected_content.append(f'version_num = {current_published_version.version.version_num}')
-                else:
-                    expected_content.append('# unpublished: no published_version_num')
-
-                for entity_version in entity.versions.all():
-                    expected_content.append(f'title = "{entity_version.title}"')
-                    expected_content.append(f'uuid = "{entity_version.uuid}"')
-                    expected_content.append(f'version_num = {entity_version.version_num}')
-                    expected_content.append('[version.container]')
-                    expected_content.append('[version.container.unit]')
-
-                self.check_toml_file(
-                    zip_path,
-                    Path(f"entities/{entity.key}.toml"),
-                    expected_content
-                )
+                if hasattr(entity, 'component'):
```

Comment:
Nit: Try to avoid doing internal logic in tests where possible. If there is a list of test entities that you expect to be components, it's better to actually list them explicitly, to reduce the odds that bad or changed app logic could cause this test code to accidentally be skipped over.